ios/Video/RCTVideoPlayerViewController.m

```diff
@@ -28,7 +28,8 @@ - (UIInterfaceOrientationMask)supportedInterfaceOrientations {
 
 - (UIInterfaceOrientation)preferredInterfaceOrientationForPresentation {
   if ([self.preferredOrientation.lowercaseString isEqualToString:@"landscape"]) {
-    return UIInterfaceOrientationLandscapeRight;
+    UIDeviceOrientation currentOrientation = [[UIDevice currentDevice] orientation];
```

Comment:
Not sure but I think you can just return `UIInterfaceOrientationLandscapeRight | UIInterfaceOrientationLandscapeLeft` no?

Reply:
Yep, tested on simulator and working fine. I am going to update PR